brx/README.markdown

```diff
@@ -0,0 +1,12 @@
+# Web Console Browser Extensions
```

Comment:
Can we rename the `brx` folder to `chrome` so its easier to guess what it contains?